src/tool/tool.go

```diff
@@ -58,7 +58,7 @@ func matchingTools(config *core.Configuration, prefix string) map[string]string
 		"jarcat":      config.Java.JarCatTool,
 		"javacworker": config.Java.JavacWorker,
 		"junitrunner": config.Java.JUnitRunner,
-		"langserver":  path.Join(config.Please.Location, "build_langserver"),
```

Comment:
can you leave the old one in too please? at least for now, otherwise anyone using it now will break.

Reply:
sure! Done!